.circleci/config.yml

```
@@ -1168,6 +1168,71 @@ jobs:
                       make postman
             - notify-on-failure
 
+    publish_rpm_packages:
+        parameters:
+            dry_run:
+                default: true
+                type: boolean
+            graviteeio_version:
+                type: string
+                default: $GRAVITEEIO_VERSION
+        
+        machine:
+            image: ubuntu-2004:202010-01
+            resource_class: medium
+        environment:
+            GRAVITEEIO_VERSION: << parameters.graviteeio_version >>
+        steps:
+            - checkout
+            - keeper/env-export:
+                  secret-url: keeper://IZd-yvsMopfQEa_0j1SDvg/field/login
+                  var-name: GIT_USER_NAME
+            - keeper/env-export:
+                  secret-url: keeper://IZd-yvsMopfQEa_0j1SDvg/custom_field/email
+                  var-name: GIT_USER_EMAIL
+            - run:
+                  name: Git config
+                  command: |
+                      git config --global user.name "${GIT_USER_NAME}"
+                      git config --global user.email "${GIT_USER_EMAIL}"
+            - run:
+                  name: "checking release version"
+                  command: |
+                    export MAJOR_VERSION_NUMBER=$(echo ${GRAVITEEIO_VERSION} | awk -F '.' '{print $1}')
+                    if [ "${MAJOR_VERSION_NUMBER}" == "1" ]; then
+                        echo "The [${GRAVITEEIO_VERSION}] is a [1.x.y] version of Gravitee.io APIM, So skipping this script made for v3"
+                        exit 0
+                    else
+                        if [ "${MAJOR_VERSION_NUMBER}" == "3" ]; then
+                            echo "The [${GRAVITEEIO_VERSION}] is a [3.x.y] version of Gravitee.io APIM"
+                        else
+                            echo "The [${GRAVITEEIO_VERSION}] is neither a [3.x.y], nor a [1.x.y] version of Gravitee.io"
+                            echo "Which is not supported yet"
+                            exit 8
+                        fi;
+                    fi;
+            - run:
+                  name: "Publishing rpms"
+                  command: |
+                        ssh -T git@github.com || true
+
+                        export GIT_GRAVITEE_PACKAGES_REPO=$(mktemp -d -t "git_gravitee_packages_repo-XXXXXXXXXX")
```

Let's simplify that name, this should be ok (I guess):
```suggestion
                        export GIT_GRAVITEE_PACKAGES_REPO=$(mktemp -d)
```

---

done